app/controllers/content_blocks_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ContentBlocksController < ApplicationController
+  load_and_authorize_resource except: :index
+  before_action :load_featured_researchers, only: :index
+  authorize_resource only: :index
+
+  def index
+  end
+
+  def destroy
+    @researcher = ContentBlock.find(params[:id])
+    if @researcher.destroy
+      flash[:success] = "Sucessfully deleted."
+    else
+      flash[:error] = "Error in deleting entry."
```

Comment:
I think it'd be good to be a little more specific with the error, maybe "Error while deleting Featured Researcher."